File: develop/client/KatachiShootingStarOnline/Assets/Scripts/Model/Data/TypingWord.cs
using System.Collections;
using System.Collections.Generic;
using System.Linq;
using System.Text;

/// <summary>
/// タイピング用言葉クラス
/// </summary>
public class TypingWord : Word
{
    private string roman;
    public string Roman { get => roman; }
    public TypingWord(string label, string roman)
        : base(label)
    {
        this.roman = roman;
    }
}

File: develop/server/TypingStarOnlineServer/src/main/java/app/katachiplus/domain/model/TypingWord.java
package app.katachiplus.domain.model;

import lombok.AllArgsConstructor;
import lombok.Data;

/**
 * タイピングワード
 * */
@Data
@AllArgsConstructor
public class TypingWord {
	private String label;
	private String roman;
	
	public static TypingWord fromCSV(String line) {
		var word = line.split(",");
		return new TypingWord(word[0], word[1]);
	}
}
